### PR TITLE
Fix test of lr scheduler in graph mode

### PR DESCRIPTION
**Pull request by leaves-zwx**
根据 https://github.com/Oneflow-Inc/OneTeam/issues/1039 修复 test_graph_lr_scheduler.py 概率失败的问题。

### CODE REVIEW

File: python/oneflow/test/graph/test_graph_lr_scheduler.py [hunk 1]
```diff
@@ -107,12 +107,13 @@ def _compare_graph_lr_scheduler_with_eager(test_case, **kwargs):
     graph = MyGraph(module, optimizer, lr_scheduler)
 
     with _DebugMode():
-        for _ in range(iters):
+        for _ in range(iters + 1):
             ret = graph(_rand_input())
             ret.numpy()  # sync for graph finishing
 
     lr_log_file = glob.glob("log/**/train_step2lr.csv", recursive=True)[0]
```
Comment (jackalcooper) [inline python/oneflow/test/graph/test_graph_lr_scheduler.py:114]: 这个代码每次运行如果log文件多挺影响性能？

**Review by jackalcooper — APPROVED** [review]

**Review by chengtbf — APPROVED** [review]

File: python/oneflow/test/graph/test_graph_lr_scheduler.py (hunk 1 above)
Comment (leaves-zwx) [reply python/oneflow/test/graph/test_graph_lr_scheduler.py:114]: 是不是应该改成 lr_log_file = glob.glob("log/*/train_step2lr.csv", recursive=False) ? 

这里 log 文件应该只包含当前单测的内容？为什么说 log 文件会多呢？

Comment (jackalcooper) [reply python/oneflow/test/graph/test_graph_lr_scheduler.py:114]: 是我想太多了，我担心的是这个脚本有可能一直运行，log目录会积累很多次运行产生的日志。话说如果是个CI里面的脚本，是不是可以除了保存是不是也可以直接打印

Comment (leaves-zwx) [reply python/oneflow/test/graph/test_graph_lr_scheduler.py:114]: 但这个往 train_step2lr.csv 写 lr 是 lr scheduler kernel 在 debug mode 下的原生行为，这个保存不是我在测试用例里面写的，我只是把它拿出来对比而已，现在没有简单的方式访问 graph 中的 lr。还有你说的 ”直接打印“ 是什么意思？

File: python/oneflow/test/tensor/test_tensor_part_2.py [hunk 2]
```diff
@@ -391,6 +391,10 @@ def test_floor_tensor_with_random_data(test_case):
         y = x.floor()
         return y
 
+    @unittest.skip(
+        "TODO: probably fail, skip for now and fix it in future."
+        "ref to: https://github.com/Oneflow-Inc/OneTeam/issues/1006#issuecomment-1022768858"
+    )
```
Comment (leaves-zwx) [inline python/oneflow/test/tensor/test_tensor_part_2.py:397]: 这里先跳过了这个会概率出错的 test case

File: python/oneflow/test/tensor/test_tensor_part_2.py [hunk 3]
```diff
@@ -110,23 +107,20 @@ def _test_sin_tensor_function(test_case, shape, device):
...
 
-    @flow.unittest.skip_unless_1n1d()
     def test_std_tensor_function(test_case):
         np_arr = np.random.randn(9, 8, 7, 6)
         input = flow.Tensor(np_arr)
         of_out = input.std(dim=1, unbiased=False, keepdim=False)
         np_out = np.std(np_arr, axis=1)
         test_case.assertTrue(np.allclose(of_out.numpy(), np_out, 1e-04, 1e-04))
 
-    @flow.unittest.skip_unless_1n1d()
```
Comment (leaves-zwx) [inline python/oneflow/test/tensor/test_tensor_part_2.py:129]: 去除重复的判断，这个条件在类 TestTensor 上已经做了限定

File: python/oneflow/test/tensor/test_tensor_part_2.py [hunk 4]
```diff
@@ -359,41 +335,40 @@ def test_clip_inplace_maxnone_tensor_with_random_data(test_case):
...
         return y
 
-    @flow.unittest.skip_unless_1n1d()
+    @unittest.skip(
+        "TODO: probably fail, skip for now and fix it in future."
+        "ref to: https://github.com/Oneflow-Inc/OneTeam/issues/1006#issuecomment-1022768858"
+    )
     @autotest(check_graph=True)
-    def test_tesnor_var_all_dim_with_random_data(test_case):
+    def test_tensor_var_all_dim_with_random_data(test_case):
```
Comment (leaves-zwx) [inline python/oneflow/test/tensor/test_tensor_part_2.py:371]: 修复错误的拼写
